products/windowsEmbedded.md

```diff
@@ -1,40 +1,57 @@
 ---
 title: Microsoft Windows Embedded
-permalink: /windowsembedded
+category: os
 iconSlug: windows
+permalink: /windows-embedded
+alternate_urls:
+- /windowsembedded
+versionCommand: winver
 releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Windows%20Embedded
-category: os
-activeSupportColumn: true
+releaseLabel: "Windows Embedded __RELEASE_CYCLE__"
```

Comment:
```suggestion
```
All entries in the table have the same prefix, we should remove this on this occasion. Ok?

Reply:
Yep !